.github/workflows/linux-build-and-test.yaml

```diff
@@ -40,11 +40,11 @@ jobs:
         sudo apt-get update
         sudo apt-get install flex bison lcov systemd-coredump gdb libipc-run-perl libtest-most-perl ${{ matrix.extra_packages }}
 
-    - name: Install macOS Dependencies using perl 5.18
+    - name: Install macOS Dependencies using Perl
       if: runner.os == 'macOS'
       run: |
-        sudo perl5.18 -MCPAN -e "CPAN::Shell->notest('install', 'IPC::Run')"
-        sudo perl5.18 -MCPAN -e "CPAN::Shell->notest('install', 'Test::Most')"
+        sudo perl -MCPAN -e "CPAN::Shell->notest('install', 'IPC::Run')"
```

Comment:
I used `5.18` because otherwise there are weird @INC issues for these two modules on Mac.

Reply:
Do you need to change the check above to 5.18 instead of 5.8?

Reply:
@nikkhils CI doesn't report any weird issues, so I guess this change is fine?

Reply:
all good now for me